packages/tomls/src/omnibus/reya_network.toml

```diff
@@ -12,6 +12,7 @@ include = [
     "../ranks/mainnet.toml",
     "../collateral_pools/collateral_pool_1/mainnet.toml",
     "../orders_gateway/mainnet.toml",
+    "../oracle_adapters/mainnet.toml",
```

Comment:
let's keep it out of reya_network for now and focus on reya_cronos

Reply:
https://github.com/Reya-Labs/reya-deployments/pull/54/commits/3e556ae99ff04c88cd8369b6e2c53db88f1a10e6